operators/advanced_network/advanced_network/managers/rivermax/rivermax_service/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.20)
+
+project(rivermax_service)
+
+add_library(${PROJECT_NAME} SHARED
+  #rmax_base_service.cpp
+  #rmax_ipo_receiver_service.cpp
+  #ipo_receiver_io_node.cpp
+  # We'll add the rmax_ipo_receiver.cpp after fetching it
+)
+
+target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_compile_features(${PROJECT_NAME}
+  PUBLIC cxx_std_17
+)
+target_compile_definitions(${PROJECT_NAME} PUBLIC RMAX_APPS_LIB_FLAT_STRUCTURE)
+
+set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
meant to use a glob variable? It will affect what's built in this folder only, so just the RDK from my understanding. can also be set as a target property instead.